src/openrct2-ui/windows/Error.cpp

```diff
@@ -99,8 +100,8 @@ class ErrorWindow final : public Window
 
     void OnUnknown5() override
```

Comment:
There is no code that ever calls this function currently, if I'm not mistaken this is supposed to close after 8 seconds on its own so this is best moved to OnPeriodicUpdate which will be actually called every second after #20438 is merged.

Reply:
Yes i noticed that as well.

Reply:
Now its merged and you can change the function, I also believe that this is an actual bug that the errors never go away on their on so this could also need a changelog entry.

Reply:
Okay done.